### PR TITLE
[4.1] Task Schedule Long description & more

### PR DESCRIPTION
**Pull request by brianteeman**
Adds the code to handle long descriptions in the same way as with modules - satisfies a @todo

Removes "Are these of use here?" as they're not

Removes "task-params" from the ignore fieldsets. It should not be part of the ignore fieldsets here and only _worked_ because it was actually a typo and should have been "task_params"

### CODE REVIEW

File: administrator/components/com_scheduler/tmpl/task/edit.php [hunk 1]
```diff
@@ -48,10 +47,6 @@
 	$this->ignore_fieldsets[] = $fieldset->name;
 endforeach;
 
-// ? : Are these of use here?
-$isModal = $input->get('layout') === 'modal';
-$layout = $isModal ? 'modal' : 'edit';
-$tmpl = $isModal || $input->get('tmpl', '') === 'component' ? '&tmpl=component' : '';
```
Comment (Quy) [inline administrator/components/com_scheduler/tmpl/task/edit.php:55]: Variables utilize in line 52.

File: administrator/components/com_scheduler/tmpl/task/edit.php [hunk 2]
```diff
@@ -32,13 +32,12 @@
...
 
 // Fieldsets to be ignored by the `joomla.edit.params` template.
-$this->ignore_fieldsets = ['aside', 'details', 'exec_hist', 'custom-cron-rules', 'basic', 'advanced', 'priority', 'task-params'];
+$this->ignore_fieldsets = ['aside', 'details', 'exec_hist', 'custom-cron-rules', 'basic', 'advanced', 'priority'];
 
 // Used by the `joomla.edit.params` template to render the right template for UI tabs.
 $this->useCoreUI = true;
 
 $advancedFieldsets = $this->form->getFieldsets('params');
-
```
Comment (richard67) [inline administrator/components/com_scheduler/tmpl/task/edit.php]: I think the empty line 41 should not be removed.

**Comment by Quy** [comment]
Clicking `Show full description ...` results in browser's console:

> Uncaught TypeError: document.querySelector(...) is null

**Comment by brianteeman** [comment]
@quy its the same code as for the modules when they have long descriptions. Does it work there?

**Comment by Quy** [comment]
No. It is also happening in v4.0 so please ignore my report in this PR.

File: administrator/components/com_scheduler/tmpl/task/edit.php [hunk 3]
```diff
@@ -85,13 +80,35 @@ class="form-validate">
...
+									{
+										$long_description = '';
+									}
+								}
+							}
+						?>
+						<p><?php echo $short_description; ?></p>
+						<?php if ($long_description) : ?>
+							<p class="readmore">
+								<a href="#" onclick="document.querySelector('#tab-description').click();">
```
Comment (Quy) [inline administrator/components/com_scheduler/tmpl/task/edit.php]: Please change per PR #36683.

Comment (brianteeman) [reply administrator/components/com_scheduler/tmpl/task/edit.php]: done

**Comment by brianteeman** [comment]
All requested changes have been made

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9f4622f2efd0de2a719f8d85b2a6894adfc3c08e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36519">issues.joomla.org/tracker/joomla-cms/36519</a>.</sub>

**Comment by ditsuke** [comment]
I have tested this item :white_check_mark: successfully on 9f4622f2efd0de2a719f8d85b2a6894adfc3c08e<br /><br />Works as advertised and nothing breaks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36519">issues.joomla.org/tracker/joomla-cms/36519</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36519">issues.joomla.org/tracker/joomla-cms/36519</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**